py-polars/polars/dataframe/plotting.py

```diff
@@ -77,8 +76,6 @@ def bar(
             Column with y-coordinates of bars.
         color
             Column to color bars by.
-        tooltip
-            Columns to show values of when hovering over bars with pointer.
         **kwargs
             Additional keyword arguments passed to Altair.
```

Comment:
note that this is a backwards-compatible change, as anyone passing `tooltip` will still have their code work due to `**kwargs`